doc/source/data/loading-data.rst

```diff
@@ -531,6 +531,34 @@ Ray Data interoperates with distributed data processing frameworks like
             {'col1': 1, 'col2': '1'}
             {'col1': 2, 'col2': '2'}
 
+    .. tab-item:: Iceberg
+
+        To create a :class:`~ray.data.dataset.Dataset` from an `Iceberg Table
+        <https://iceberg.apache.org>`__,
+        call :func:`~ray.data.read_iceberg`. This function creates a ``Dataset`` backed by
+        the distributed files that underlie the Iceberg table.
+
+        ..
+            TODO: This code snippet might not work correctly. We should test it.
+
+        .. testcode::
+            :skipif: True
```

Comment:
i think we still need it since this is just a test table demonstrating usage.

Reply:
Yup, this is hard to test because it's a glue example etc., and because we need an iceberg table - so I followed the example of the lance tables and skipped this. However, there are other iceberg tests in the PR!